app/scripts/views/settings/devices.js

```diff
@@ -42,9 +69,15 @@ define(function (require, exports, module) {
       devices.on('remove', this._onDeviceRemoved.bind(this));
     },
 
+    _formatDevicesList: function (devices) {
+      return _.each(devices, function (device) {
+        device.lastAccessTime = formatConnectedTimestamp(Number(device.lastAccessTime));
```

Comment:
Where do we stand on referential transparency? Just wondering whether this function would be preferable if it returned `_.map` and we ditched the internal assignment.